webapp/templates/analyzer.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-{{ template "_head_common.html" }}
+{{ template "_head_common.html" . }}
```

Comment:
Below, it seems like we were already using `.Before` without the `.` here. So is this necessary, or good practice, or ...?

Reply:
This is to pass the current dot (we are at the top level here, so this is the whole template data) into `_head_common.html` since it now requires data (`.User`). Previously `_head_common.html` didn't need any data, and the default (no argument) sets dot to `nil` in the included template.

This clause does not affect the rest of the current template.